README.md

```diff
@@ -13,7 +13,7 @@ npm install -g asyncapi-generator
 Or use all the commands below using Docker by prefixing them with:
 
 ```bash
-docker run --rm -it -v $PWD:/app -w /app asyncapi/generator [ag|anc] [COMMAND HERE]
+docker run --rm -it -v $PWD:/app -w /app asyncapi/generator ag [COMMAND HERE]
```

Comment:
Could probably just have it call `ag` by default so user doesn't have to put it here.

Reply:
You're right. That would be better.